project/plugins.sbt

```diff
@@ -0,0 +1,5 @@
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
+
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.8.3")
```

Comment:
Could try upgrading to 0.9.1 to use the new Riff Raff deployment type (see Phil W's email)